File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/company/dtos/PatchCompanyResponse.java
package com.slotify.backend.spring.company.dtos;

import lombok.Builder;
import lombok.Getter;
import lombok.ToString;

import java.util.UUID;

@Builder
@Getter
@ToString
public class PatchCompanyResponse {

    private UUID companyId;
    private Integer defaultMaxConcurrentServices;
    private String companyName;
    private String phoneNumber;
    private String physicalAddress;
    private String s3ImageUrl;
    private String description;
}

File: frontend/app/src/main/java/com/raj/slotify/dtos/company/PatchCompanyResponse.kt
package com.raj.slotify.dtos.company

import com.google.gson.annotations.SerializedName
import java.util.UUID

data class PatchCompanyResponse(
    @SerializedName("companyId")
    val companyId: UUID,

    @SerializedName("defaultMaxConcurrentServices")
    val defaultMaxConcurrentServices: Int,

    @SerializedName("companyName")
    val companyName: String,

    @SerializedName("phoneNumber")
    val phoneNumber: String,

    @SerializedName("physicalAddress")
    val physicalAddress: String,

    @SerializedName("s3ImageUrl")
    val s3ImageUrl: String?,

    @SerializedName("description")
    val description: String?
)
